pkg/surveyext/editor.go

```diff
@@ -18,25 +18,34 @@ import (
 )
 
 var (
-	bom    = []byte{0xef, 0xbb, 0xbf}
-	editor = "nano" // EXTENDED to switch from vim as a default editor
+	bom           = []byte{0xef, 0xbb, 0xbf}
+	defaultEditor = "nano" // EXTENDED to switch from vim as a default editor
 )
 
 func init() {
 	if runtime.GOOS == "windows" {
-		editor = "notepad"
+		defaultEditor = "notepad"
 	} else if g := os.Getenv("GIT_EDITOR"); g != "" {
-		editor = g
+		defaultEditor = g
 	} else if v := os.Getenv("VISUAL"); v != "" {
-		editor = v
+		defaultEditor = v
 	} else if e := os.Getenv("EDITOR"); e != "" {
-		editor = e
+		defaultEditor = e
 	}
 }
 
 // EXTENDED to enable different prompting behavior
 type GhEditor struct {
 	*survey.Editor
+	EditorName string
```

Comment:
Ditto: this is called `EditorName` but can contain the full invocation line which is latter shellsplit. In name it's similar to `EditorTemplateData.EditorName`, but not in behavior. Therefore I would suggest a different name to help clarify the difference